### ISSUE TITLE
 contextMenuService not showing the menus in angular 11

### ISSUE THREAD

**Issue by bala-visolve** [isaacplmann/ngx-contextmenu#237]
I have updated my application from angular 8 to 11.

ngx-contextmenu is not showing the menu through,contextMenuService .

`        this.contextMenuService.show.next({`

I have updated the packages version to recent also but still context menu is not showing up.

Any help would be really appreciated!!

Angular: 11
CDK/Material: ngx-contextmenu@6.0.0
Browser(s): Chrome and firefox
Operating System ( Ubuntu):

**Comment by rrauenza** [comment]
This and #236 seem to be related to https://stackoverflow.com/questions/60432992/expression-has-changed-after-it-was-checked-using-viewchild-reference

I haven’t been able to reproduce outside of our app, but it seems to be related to detect changes not detecting changes as well as possibly a similar error message mentioned in the SO link.

when it works, the menu shows after ngx contextmenu calls contextMenuContent.changeDetectorRef.detectChanges() and when it doesn’t, this call seems to do nothing.

**Comment by rrauenza** [comment]
> This and #236 seem to be related to https://stackoverflow.com/questions/60432992/expression-has-changed-after-it-was-checked-using-viewchild-reference

No, doing ViewChild with static=true seems to fix the error message, but not the menu coming up.

**Comment by rrauenza** [comment]
We were using too old a version of angular material.  Upgrade fixed it.

**Comment by chitgoks** [comment]
what version did you used for angular 11?
